### PR TITLE
Manually kickstart GC for android after navigation

### PR DESCRIPTION
**Pull request by PanayotCankov**

### CODE REVIEW

File: tns-core-modules/ui/frame/frame.android.ts [hunk 1]
```diff
@@ -261,6 +275,8 @@ export class Frame extends frameCommon.Frame {
         if (trace.enabled) {
             trace.write(`END TRANSACTION ${fragmentTransaction}`, trace.categories.Navigation);
         }
+
+        android.os.Looper.myQueue().addIdleHandler(getIdleHandler());
```
Comment (hshristov) [inline tns-core-modules/ui/frame/frame.android.ts]: Move this from here to _navigateCore & _goBackCore

File: tns-core-modules/ui/frame/frame.android.ts [hunk 2]
```diff
@@ -292,6 +308,8 @@ export class Frame extends frameCommon.Frame {
             // this handles cases where user may navigate to an inner page without adding it on the backstack
             manager.popBackStack(backstackEntry.fragmentTag, android.app.FragmentManager.POP_BACK_STACK_INCLUSIVE);
         }
+
+        android.os.Looper.myQueue().addIdleHandler(getIdleHandler());
```
Comment (hshristov) [inline tns-core-modules/ui/frame/frame.android.ts]: Move this from here to _navigateCore & _goBackCore

**Comment by hshristov** [comment]
run ci

**Comment by vchimev** [comment]
Currently, the app crashes trying to execute the transitions tests.

**Comment by vchimev** [comment]
This will help in https://github.com/NativeScript/NativeScript/issues/2580.

**Pull request merged**